roles/common/tasks/crd_tests.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Verify logging CRD "{{ item }}"
```

Comment:
Please remove "feature name" from task name as these are common tests for multiple jobs and feature. Suggest:

- name: Verify CRD "{{ item"